app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversation.kt

```diff
@@ -0,0 +1,123 @@
+package com.wire.android.ui.home.newconversation
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.wire.android.R
+import com.wire.android.model.UserStatus
+import com.wire.android.ui.common.NavigationIconType
+import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.textfield.WirePrimaryButton
+import com.wire.android.ui.home.conversationslist.common.RowItem
+import com.wire.android.ui.home.conversationslist.folderWithElements
+import com.wire.android.ui.theme.wireTypography
+
+
+@Composable
+fun NewConversationScreen(newConversationViewModel: NewConversationViewModel = hiltViewModel()) {
+    val state by newConversationViewModel.newConversationState
+
+    NewConversationContent(
+        state = state,
+        onCloseClick = { newConversationViewModel.close() }
+    )
+}
+
+@Composable
+fun NewConversationContent(
+    state: NewConversationState,
+    onCloseClick: () -> Unit
+) {
+    val lazyListState = rememberLazyListState()
+
+    Box {
+        Column(
+            Modifier
+                .fillMaxWidth()
+                .padding(top = dimensions().topBarSearchFieldHeight)
+                .wrapContentSize()
+        ) {
+            LazyColumn(
+                state = lazyListState,
+                modifier = Modifier.weight(1f),
+                contentPadding = PaddingValues(top = dimensions().topBarSearchFieldHeight)
+            ) {
+                folderWithElements(
+                    header = { stringResource(R.string.label_contacts) },
+                    items = state.contacts
+                ) { contact ->
+                    ContactItem(
+                        contact.name,
+                        contact.userStatus,
+                        contact.avatarUrl
+                    )
+                }
+            }
+            Divider()
+            Column(modifier = Modifier.padding(all = 16.dp)) {
```

Comment:
it's always debatable if a specific size will be changed, but it's probably better to always use `wireDimensions` and at least use `spacing_x`.